third_party/closure-compiler/runner/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,34 @@
+import java.util.List;
+
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+
+  private static final List<String> LOG_STRIP_SUFFIX = ImmutableList.of(
+      "dev.fine",
+      "dev.assert");
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
+    options.setStripTypes(Sets.newHashSet(LOG_STRIP_SUFFIX));
```

Comment:
This is crazy inconvenient. Ideally we'd find a way to do stripping before renaming/obfuscation. No idea why they thought it'd be a good idea. A stopgap option might be to disable dev/user obfuscation? Obviously that's a bit annoying too, but since we considering to doing them as globals, it doesn't really matter. And most of uses of these APIs are `dev.fine` and `dev.assert` - so it might be ok anyway.